process/economics/testEconomicsData.go

```diff
@@ -12,6 +12,7 @@ type TestEconomicsData struct {
 // SetMaxGasLimitPerBlock sets the maximum gas limit allowed per one block
 func (ted *TestEconomicsData) SetMaxGasLimitPerBlock(maxGasLimitPerBlock uint64) {
```

Comment:
Maybe you can add another parameter to this method maxGasLimitPerMetaBlock uint64, so in this way we can test with different values if we want this. Or better to create another method to SetMaxGasLimitPerMetaBlock

Reply:
it is not used rigth now - in any test it is only one value. there is no need. When there will be a test to especially change these values - it will be created.